### ISSUE TITLE
monika: cgi-bin directory is unvalid on Debian Jessie

### ISSUE THREAD

**Issue by npf** [oar-team/oar#56]
```
# tail -f /var/log/apache2/error.log
[Tue Jul 28 17:26:00.161324 2015] [cgi:error] [pid 4439] [client 194.199.27.205:33361] AH01215: suexec policy violation: see suexec log for more details
[Tue Jul 28 17:26:00.161404 2015] [cgi:error] [pid 4439] [client 194.199.27.205:33361] End of script output before headers: monika.cg

# tail -f /var/log/apache2/suexec.log
[2015-07-28 17:26:00]: uid: (108/oar) gid: (113/oar) cmd: monika.cgi
[2015-07-28 17:26:00]: command not in docroot (/usr/lib/cgi-bin/monika.cgi)
```

**Comment by npf** [comment]
@bzizou: you already changed it for oar-api in commit 06ddd467cf07b0d3ae53e4febda98e90d93d007 . However, I think it might be wiser to not change the install script but change the CGIDIR variable at install time.

**Comment by bzizou** [comment]
Ok, I see. Let's fix this together tomorrow :-)

**Comment by npf** [comment]
Hum, 
looks like monika could just run with the Apache cgi or cgid module.

they are actually the default handlers for .cgi files, and accept scripts in /usr/lib/cgi-bin
(see /etc/apache2/conf-enabled/serve-cgi-bin.conf)

/etc/oar/monika.conf just need to be owned by www-data (and 600).
Looks like it was the case before. 

I don't see a real need to run monika as oar via suexec.

if OAR API requires fastcgi, then it should be suffixed .fcgi. Then Apache fastcgi module would handle it.

**Comment by npf** [comment]
oar-vagrant should be the reference for the setup, not oar-docker which is customized to much.

**Comment by npf** [comment]
fixed in oar 2.5.5: Monika and OAR api cannot run in the same vhost.
